### ISSUE TITLE
`neogit` layer

### ISSUE THREAD

**Issue by foxyseta** [visimp/visimp#131]
Beside the `fugitive` layer, we could also consider the `neogit` one: https://github.com/NeogitOrg/neogit

**Comment by foxyseta** [comment]
As in, deprecate the old layer?

**Comment by lucat1** [comment]
Yeah. I think it would be good to phase out any non-lua plugins honestly.

OT: Also, we should make a release right now and set a timer in like 3 months to remove all deprecated layers.

**Comment by lucat1** [comment]
Yeah I would get started right now with the new release, we have some deprecated layers to remove already.

**Comment by foxyseta** [comment]
Will have this done by today.
